nipoppy/data/examples/sample_pipelines/rabies-0.5.1/invocation-analysis.json

```
@@ -0,0 +1,8 @@
+{
+    "plugin": "Linear",
+    "local_threads": 24,
+    "data_diagnosis": false,
+    "force": true,
+    "input_dir": "[[NIPOPPY_DPATH_PIPELINE_OUTPUT]]",
+    "output_dir": "[[[NIPOPPY_DPATH_PIPELINE_OUTPUT]]"
```

There is an extra opening square bracket here that should be removed:
```suggestion
    "output_dir": "[[NIPOPPY_DPATH_PIPELINE_OUTPUT]]"
```